examples/index.html

```diff
@@ -249,7 +249,7 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 
 			var link = links[ file ];
 			var name = getName( file );
-			if ( file in tags ) file += tags[ file ].join( ' ' );
+			if ( file in tags ) file += tags[ file ];
```

Comment:
This change should not be necessary.

Reply:
I just dumped a old version. I am a lazy cookie.

Reply:
By the way, I am also added webgl_loader_ttf with "text, font"

Reply:
It seems this change still needs to be undone, no?

Reply:
With this approach I think if you search for `,` you'll get a lot of results?
I think I prefer the array approach.

Reply:
Doesn’t see why, sorry. Splitting into tags and then join it again in code?
Looks like cargo cult to Tags :D . The same thing with ` `  will happens after .join().
Real problem not here. In UX probably. Nobody searching `,`.

Reply:
Fixed it. You forcing me to make a drama out of it.